src/Security/MemberAuthenticator/CMSLoginHandler.php

```diff
@@ -1,27 +1,29 @@
 <?php
 
-namespace SilverStripe\Security;
+namespace SilverStripe\Security\MemberAuthenticator;
 
 use SilverStripe\Control\HTTPResponse;
 use SilverStripe\Core\Convert;
+use SilverStripe\Security\CMSSecurity;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
 
-class CMSMemberLoginHandler extends MemberLoginHandler
+class CMSLoginHandler extends LoginHandler
 {
+    private static $allowed_actions = [
+        'LoginForm'
```

Comment:
I'm leaving in the case mismatch with PSR-2 for now, as it's a whole different level of fixing.